__tests__/app.test.js

```diff
@@ -77,4 +77,100 @@ describe('GET /api/topics', () => {
             });
         });
     });
+});
+
+describe('GET /api/articles/:article_id', () => {
+    test('response data should be on an "article" key with a 200 http status code', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toHaveProperty('article');
+        });
+    });
+
+    test('each article should have correct object layout', () => {
```

Comment:
We are only getting one object back on this endpoint - so we could be more accurate with the test description. Also as we are using a test database we know exactly what this data will look like so we could be more specific and hard code this expect response.